test/services/teams/hooks/helpers.test.js

```diff
@@ -57,9 +50,9 @@ describe('hook helpers', () => {
 				{ schoolId, role, userId: ObjectId() },
 				{ schoolId, role, userId: ObjectId() },
 			];
-			userList.push((new teamUserModel(data[0]))._doc);
-			userList.push((new teamUserModel(data[1]))._doc);
-			userList.push((new teamUserModel(data[2]))._doc);
+			userList.push(new teamUserModel(data[0])._doc);
+			userList.push(new teamUserModel(data[1])._doc);
```

Comment:
Codacy found an issue: [A constructor name should not start with a lowercase letter.](https://app.codacy.com/gh/hpi-schul-cloud/schulcloud-server/pullRequest?prid=6045788)